### PR TITLE
Install async-aws sdk

### PR DESCRIPTION
**Pull request by Nyholm**
This was missing from #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -308,34 +308,14 @@ For example to customize the SQS client:
...
-                region: us-east-1
-                credentials: false
-                # Mock Guzzle for the tests
-                handler: '@mock_handler'
-    mock_handler:
-        class: Aws\MockHandler
-        public: true
-```
-
-You can read more [in the official documentation of the SDK](https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_handlers-and-middleware.html).
```
Comment (mnapoli) [inline README.md:337]: How can users mock the client for their tests? Is that possible?

Comment (Nyholm) [reply README.md:337]: They just mock it as a normal class. There is nothing special or fancy like the official SDK. 

See https://github.com/brefphp/symfony-messenger/pull/25/files#diff-32d8454b70ebd9497ea61ebde7ffc540R40-R58

Comment (mnapoli) [reply README.md:337]: Understood, thanks!

**Comment by Nyholm** [comment]
Thank you for merging.

**Pull request merged**